### PR TITLE
Pradeep/po upload

### PR DESCRIPTION
**Pull request by pradeepko**
# Description

Add samples to showcase how to use po upload and retrieve the uploaded document.

## Related issue

Kindly link any related issues (e.g. Fixes #xyz).

### CODE REVIEW

File: sdk/SdkSamples/Orders/DownloadPoDocuments.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="UploadPoDocuments.cs" company="Microsoft">
```
Comment (kaminasy) [inline sdk/SdkSamples/Orders/DownloadPoDocuments.cs:2]: DownloadPoDocuments

File: sdk/SdkSamples/Orders/DownloadPoDocuments.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.Store.PartnerCenter.Samples.Orders
+{
+    using Microsoft.Store.PartnerCenter.Models.Orders;
```
Comment (kaminasy) [inline sdk/SdkSamples/Orders/DownloadPoDocuments.cs:9]: Please rearrange the namespaces

File: sdk/SdkSamples/Orders/DownloadPoDocuments.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    using System.Net.Http;
+    using System.Windows.Forms;
+
+    /// <summary>
+    /// A scenario that updates a customer order by uploading PO attachments.
+    /// </summary>
+    public class DownloadPoDocuments : BasePartnerScenario
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UploadPoDocuments"/> class.
```
Comment (kaminasy) [inline sdk/SdkSamples/Orders/DownloadPoDocuments.cs:22]: DownloadPoDocuments

File: sdk/SdkSamples/Orders/DownloadPoDocuments.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+            string customerId = this.ObtainCustomerId("Enter the ID of the customer whom to retrieve their orders");
+            string orderId = this.ObtainOrderID("Enter the ID of order to retrieve");
+            string attachmentId = this.ObtainOrderID("Enter the ID of attachment to retrieve");
+
+            this.Context.ConsoleHelper.StartProgress("Retrieving attachment");
+
+            var attachment = partnerOperations.Customers.ById(customerId).Orders.ById(orderId).Attachments.ById(attachmentId).Download(); 
+            this.Context.ConsoleHelper.StopProgress();
+            this.Context.ConsoleHelper.WriteObject(attachment, "Uploaded documents to customer order");
```
Comment (kaminasy) [inline sdk/SdkSamples/Orders/DownloadPoDocuments.cs:44]: Is this the correct description?

File: sdk/SdkSamples/Orders/UploadPoDocuments.cs [hunk 5]
```diff
@@ -0,0 +1,76 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="UploadPoDocuments.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.Store.PartnerCenter.Samples.Orders
+{
+    using Microsoft.Store.PartnerCenter.Models.Orders;
```
Comment (kaminasy) [inline sdk/SdkSamples/Orders/UploadPoDocuments.cs:9]: Please rearrange namespaces. Systems on the top

**Review by kaminasy — CHANGES_REQUESTED** [review]

File: sdk/SdkSamples/Orders/DownloadPoDocuments.cs [hunk 6]
```diff
@@ -0,0 +1,40 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="DownloadPoDocuments.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.Store.PartnerCenter.Samples.Orders
+{
+    /// <summary>
+    /// A scenario that updates a customer order by uploading PO attachments.
```
Comment (kaminasy) [inline sdk/SdkSamples/Orders/DownloadPoDocuments.cs:10]: A scenario that updates a customer order by downloading PO attachments

**Review by kaminasy — CHANGES_REQUESTED** [review]

**Review by kaminasy — APPROVED** [review]

**Review by brijeshp-ms — APPROVED** [review]

**Pull request merged**